Tensorboard Web App fails to build

/kind bug
/area tensorboard

Sub issue of https://github.com/kubeflow/kubeflow/issues/5526
The Tensorboard Web App fails to build, in part due to missing dependencies which are solved by https://github.com/kubeflow/kubeflow/pull/5528. However, the following error still remains:
```
ERROR in src/app/pages/index/config.ts:73:39 - error TS2345: Argument of type 'TableColumn' is not assignable to parameter of type '{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; } | ConcatArray<{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { ...; }>'.
  Type 'TableColumn' is not assignable to type '{ matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; }'.
    Types of property 'value' are incompatible.
      Type 'PropertyValue | StatusValue | ActionIconValue | ActionListValue | DateTimeValue | TemplateValue | ChipsListValue | ComponentValue' is not assignable to type 'PropertyValue'.
        Type 'StatusValue' is missing the following properties from type 'PropertyValue': tooltipField, popoverField, truncate, isLink, and 3 more.

73   columns: tableConfig.columns.concat(actionsCol),
                                         ~~~~~~~~~~

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@0.0.0 build: `npm run copyLibAssets && ng build --prod --base-href /tensorboard/ --deploy-url /tensorboard/static/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/david/.npm/_logs/2021-01-18T12_37_05_520Z-debug.log
```
VS Code gives the following information:
```
No overload matches this call.
  Overload 1 of 2, '(...items: ConcatArray<{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; }>[]): ({ ...; } | { ...; })[]', gave the following error.
    Argument of type 'TableColumn' is not assignable to parameter of type 'ConcatArray<{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; }>'.
      Type 'TableColumn' is missing the following properties from type 'ConcatArray<{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; }>': length, join, slice
  Overload 2 of 2, '(...items: ({ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; } | ConcatArray<{ ...; } | { ...; }>)[]): ({ ...; } | { ...; })[]', gave the following error.
    Argument of type 'TableColumn' is not assignable to parameter of type '{ matHeaderCellDef: string; matColumnDef: string; value: StatusValue; } | { matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; } | ConcatArray<{ ...; } | { ...; }>'.
      Type 'TableColumn' is not assignable to type '{ matHeaderCellDef: string; matColumnDef: string; value: PropertyValue; }'.
        Types of property 'value' are incompatible.
          Type 'StatusValue | PropertyValue | ActionListValue | ActionIconValue | DateTimeValue | ChipsListValue | ComponentValue | TemplateValue' is not assignable to type 'PropertyValue'.
            Type 'StatusValue' is missing the following properties from type 'PropertyValue': tooltipField, popoverField, truncate, isLink, and 3 more.ts(2769)
```
/cc @kimwnasptd

--- a/components/crud-web-apps/tensorboards/frontend/src/app/pages/index/config.ts
+++ b/components/crud-web-apps/tensorboards/frontend/src/app/pages/index/config.ts
@@ -6,9 +6,10 @@ import {
   ActionButtonValue,
   TableColumn,
   TRUNCATE_TEXT_SIZE,
+  TableConfig,
 } from 'kubeflow';
 
-const tableConfig = {
+const tableConfig: TableConfig = {
   title: 'Tensorboards',
   newButtonText: 'NEW TENSORBOARD',
   columns: [

